tools/dind/start-resin-supervisor

```diff
@@ -2,8 +2,14 @@
 
 # Adapted from: https://github.com/balena-os/meta-balena/blob/v2.45.0/meta-balena-common/recipes-containers/resin-supervisor/resin-supervisor/start-resin-supervisor
 
+# We still need to include resin-vars on legacy systems
+if [ -f /usr/sbin/resin-vars ]; then
```

Comment:
I know this change is because of @cywang117 comment, but it shouldn't be necessary right? AFAIK this file is only used with balenaos-in-container so we should not get the case of an old OS with this new supervisor or am I wrong?

Reply:
@pipex Who would be able to confirm this?

Reply:
I'm pretty sure the file is not being used after testing with dindctl script, but I opened a thread about it on FD

Reply:
Per discussions in the thread in FD, it's been confirmed that people still use this script, and that older OS's can use newer Supervisor versions, so this line is most likely necessary to prevent errors.